experiment/measurer/measure_worker.py

```diff
@@ -79,14 +81,15 @@ def _get_bucket_cycle_state_file_path(self, cycle: int) -> str:
         state_file_path = os.path.join(self.state_dir, state_file_name)
         return exp_path.filestore(pathlib.Path(state_file_path))
 
-    def _get_previous_cycle_state(self) -> list:
+    def _get_previous_cycle_state(self, cycle_dependent=True) -> list:
         """Returns the state from the previous cycle. Returns [] if |self.cycle|
         is 1."""
         if self.cycle == 1:
             return []
 
         previous_state_file_bucket_path = (
-            self._get_bucket_cycle_state_file_path(self.cycle - 1))
+            self._get_bucket_cycle_state_file_path((
+                self.cycle - 1) if cycle_dependent else 0))
```

Comment:
Should the second parenthesis come down?
`(self.cycle - 1) ...`

Reply:
I corrected it, but `make format` does this again.

Reply:
Adding `pylint: disable-all` helps me maintain the formatting.

Reply:
removed unnecessary `pylint` annotations.